docs/Documentation/openvino_workflow.md

```diff
@@ -13,7 +13,7 @@
    
 
 | :doc:`Model Preparation <openvino_docs_model_processing_introduction>`
-| With Model Downloader and Model Optimizer guides, you will learn to download pre-trained models and convert them for use with OpenVINO™. You can use your own models or choose some from a broad selection provided in the Open Model Zoo.
+| With Model Conversion API guide, you will learn to convert pre-trained models for use with OpenVINO™. You can use your own models or choose some from a broad selection provided in the Open Model Zoo.
```

Comment:
@andrei-kochin, is it still valid to refer to OMZ?

Reply:
No, it is not valid.
@sgolebiewski-intel please replace Open Model Zoo with "TF Hub, Hugging Face, Torchvision models"

Reply:
> No, it is not valid. @sgolebiewski-intel please replace Open Model Zoo with "Tensorflow Hub, Hugging Face, Torchvision models"

```suggestion
| With Model Conversion API guide, you will learn to convert pre-trained models for use with OpenVINO™. You can use your own models or choose some from a broad selection in online databases, such as `TensorFlow Hub <https://tfhub.dev/>`__, `Hugging Face <https://huggingface.co/>`__, `Torchvision models <https://pytorch.org/hub/>`__..
```